src/models/generated/Line.jl

```diff
@@ -61,6 +65,10 @@ mutable struct Line <: ACBranch
     rating::Float64
     "Minimum and maximum angle limits (radians)"
     angle_limits::MinMax
+    "Second current rating; entered in MVA."
```

Comment:
@jd-lara Are we okay with being `Union{Nothing, Float64}`? If we default to 0.0, and only uses Float64, our code will require to check the ratings with `iszero` to bypass rather than `isnothing`. 

Nothing in here makes sense since it represents that we do not have data, while 0.0 will be something like PSSE, that 0.0 means no data.

Reply:
I am fixing this in a separate PR and branch